src/llmcompressor/transformers/sparsification/sparse_model.py

```
@@ -62,6 +62,13 @@ def from_pretrained(
             pretrained_model_name_or_path directory and applied if found
         :return the created model for causal language modeling
         """
+        logger.info(
```

I would throw an error if this is tried, not just an info.